website/docs/index.md

```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 1
+title: Documentation
```

Comment:
changing the title from "About OpenLineage" to "Documentation" is not ideal for me, we're losing context and seems a bit odd

Reply:
@JDarDagran thanks for the feedback. I moved the new title to a head tag and reverted the change to the Markdown metadata.